content/mentee-application.md

```diff
@@ -0,0 +1,83 @@
+---
+id: mentee-application
+title: Mentee Application Template
+---
+
+# Our Mentee Application Template
+
+Hello! Thank you for your interest in being a part of Galaxy Mentoring Network!
+
+**Kindly copy the content of this template and add paste it in a text editor so that you can have a draft before you tranfer your responses to the form.**
+
+# Section 1: Personal Information
+
+The information you entered in this section will be added to your profile on our website. Be free to let's know who you are!
+
+1. First Name
+2. Last Name
+3. Email
+4. Gitter/Matrix ID
+5. State/Country
+6. Pronouns
+7. Affiliations
+8. Age (in ranges:) ***Choose one option***
+    20-30
+    30-40
+    40-50
+    50 - 60
+    60-70
+    70+
+    Prefer not to say
+
+# Section 2: Background Questions
+
+This is to know a bit about how about you and how best we can support during the mentoring program.
+
+9.  Educational Level ***Choose one option***
+    High School
+    Undergraduate
+    Postprogarduate
+    Postdoctoral
+    Engineer
+    PI
+    Other
+10. Current Occupation ***Choose one option***
+    Employed
+    Self-employed
+    Freelancing
+    Student
```

Comment:
List

Reply:
And same for all other similar questions

Reply:
Done, thanks for pointing!